dmapiclient/data.py

```diff
@@ -24,12 +24,15 @@ def find_audit_events(
             earliest_for_each_object=None,
             user=None,
             data_supplier_id=None,
+            *,
```

Comment:
`*`? What else are we expecting?

Reply:
I though that `data_draft_id` should be a [keyword-only argument](https://www.python.org/dev/peps/pep-3102/)

Reply:
You could say that about all the others too (breaking changes notwithstanding) 😹 I feel like this has cropped up before but looking through old PRs I can't see it, so I might be imagining it.

Reply:
Yeah, I would've put it in earlier but it would have been a breaking change.

> I feel like this has cropped up before

I've used this before here https://github.com/alphagov/digitalmarketplace-scripts/pull/388, is this the PR you meant?

Reply:
That's the one, well remembered! 

I don't think we use this elsewhere in the API client yet. Maybe we should? If so, we should be consistent with it. Perhaps a similar to our typing approach in `dmutils` - if it's a new method (or an already breaking change to an old method), start using it? Document it somewhere?

Reply:
I think we should use it in our api client, the reason the keyword only parameters feature was added was to be able to prevent breaking changes in apis, which is obviously useful here. I'm not sure there's any point limiting it to only breaking changes though?